integration/src/test/java/org/duracloud/integration/durastore/storage/probe/ProbedRestS3Client.java

```diff
@@ -58,16 +57,16 @@
  *
  * @author Bill Branan
  */
-public class ProbedRestS3Client extends AmazonS3Client implements MetricsProbed {
+public class ProbedRestS3Client extends AbstractAmazonS3 implements MetricsProbed {
 
     private static final long serialVersionUID = 1L;
 
     protected MetricsTable metricsTable;
 
     protected Metric metric = null;
 
-    public ProbedRestS3Client(AWSCredentials credentials) throws AmazonServiceException {
-        super(credentials);
```

Comment:
Credentials still need to be passed down to allow the connection to S3 to be made for integration tests.